config/routes.rb

```diff
@@ -1,2 +1,6 @@
 Rails.application.routes.draw do
+  scope "/:locale", locale: /en/ do
+    resources :users, only: [:new]
+    resources :budgets, only: [:new]
```

Comment:
These are not needed yet, cause there aren't any controllers and views in this PR.
Also, is the local scope still needed?